_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,283 @@
+---
```

Comment:
Will you make changes discussed yesterday of linking top level doc to this sub page?

Reply:
yes